tests/bgp/test_ipv6_bgp_scale.py

```diff
@@ -483,6 +550,9 @@ def flapper(duthost, pdp, bgp_peers_info, transient_setup, flapping_count, conne
         remove_routes_with_nexthops(startup_routes, nexthops_to_remove, expected_routes)
         compressed_routes = compress_expected_routes(expected_routes)
     else:
+        if all_flap:
+            # When all neighbors/ports are flapped, adding some buffer for DUT to recover functionality
+            time.sleep(10)
```

Comment:
this one is tricky, 10s is not stable. I wonder if we can use other ways to wait for things to come back, like wait for maybe bgp session to be established and etc?